src/Camelot/Views/Main/FilesPanelView.xaml.cs

```diff
@@ -26,10 +27,14 @@ public FilesPanelView()
 
         private void SubscribeToEvents() => DataContextChanged += OnDataContextChanged;
 
-        private void OnDataContextChanged(object sender, EventArgs e) =>
+        private void OnDataContextChanged(object sender, EventArgs e)
+        {
             ViewModel.DeactivatedEvent += ViewModelOnDeactivatedEvent;
+            ViewModel.ActivatedEvent += ViewModelOnActivatedEvent;
+        }
 
         private void ViewModelOnDeactivatedEvent(object sender, EventArgs e) => ClearSelection();
+        private void ViewModelOnActivatedEvent(object sender, EventArgs e) => FilesDataGrid.Focus();
```

Comment:
Please add empty line after this one just to keep everything consistent :smiley: